.github/workflows/scripts/e2e-utils.sh

```diff
@@ -419,7 +424,7 @@ e2e_verify_predicate_metadata_v1() {
 }
 
 e2e_verify_predicate_materials_v1() {
-     _e2e_verify_query "$1" "$2" '.predicate.buildDefinition.resolvedDependencies[0]'
```

Comment:
I think this function is not used

Reply:
Oh, hmm. Fair enough seems like it's not used. I'll clean this up later.

Reply:
Created #251 to track